@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -848,6 +848,30 @@ export interface StandardTexturedTechniqueParams extends BaseStandardTechniquePa
     displacementMapProperties?: TextureProperties;
 }
 
+/**
+ * Technique used to render a terrain geometry with a texture.
+ * When using this technique, the datasource will produce texture coordinates in
+ * local tile space (i.e. [0,0] at south-west and [1,1] at north-east tile corner).
+ */
+export interface StandardTexturedTerrainTechniqueParams extends StandardTexturedTechniqueParams {
```

Comment:
Maybe just TerrainTechnique would be enough

Reply:
Makes sense. Will change.